docs/_documentations/overview.md

```diff
@@ -9,21 +9,72 @@ type: document
+When it comes to running your microservices on the cloud, Codewind enables you to build and run Docker-containerised applications on [Kubernetes](https://kubernetes.io/) container orchestration applications, including [Openshift](https://www.openshift.com/), without the need for in-depth knowledge of container technology.
 
+## How does Codewind help me develop containerised applications?
+Codewind improves your inner loop experience, enabling you to create a microservice quickly, rapidly iterate on changes and make improvements to performance thanks to the following features:
 
-# Features
+1. **Access to pre-configured containerised project templates** Codewind provides pre-configured, Docker-containerised project templates ADD URL covering several languages including [Node.js](https://nodejs.dev/), Java, Python and [Swift](https://swift.org/), and several frameworks such as Express, Spring Boot and Open Liberty ADD URLS, with more on the way. [Quickly create and deploy microservices in languages and frameworks that you're familiar with](https://www.youtube.com/watch?v=zKMggp10gq4&t=12s). Modify the pre-configured projects to develop your customised microservice.
+
+2. **Ability to add your own applications** Access the seamless container development and deployment experience on your IDE that Codewind offers for your current projects.
+
+3. **Auto-build-and-run on code changes** As you develop, Codewind *automatically* pushes your code changes to your container as efficiently as possible. 
+
+4. **Ability to toggle between auto-build and manual build settings** Have control over when you want to implement Codewind's auto-build feature.
+
+5. **Project details displayed in a user-friendly panel on your IDE** Use Codewind's Project Overview interface to instantly view the endpoint, container ID, location on disk, build and run status, and ports.
+
+6. **Access application, build and container logs and run commands from within the container without leaving your IDE** Codewind integrates with your IDE's terminal window, 'problems' and 'output' panels for a smooth test and debug experience. Additionally, you can open a shell session *inside* your container with a simple click.
+
+7. **Support for your IDE's debug features**<sup>*</sup>  Full compatiility with the debug functionality of your IDE.
+
+8. **Visualise your container application's footprint such as CPU and memory usage** <sup>*</sup> Codewind's Application Metrics Dashboard exposes your microservice's core metrics, including HTTP(S) request and throughput information, as well as CPU usage, memory footprint and heap.
+
+9. **Integrated load-testing** <sup>*</sup> Configure and run load tests against your microservices using [Codewind's performance dashboard](https://www.youtube.com/watch?v=nfJt3f5TUvc). Use the graphs to compare your repeated load test results and iterate on performance improvements. 
+
```

Comment:
Suggestions for numbered list:
- A numbered list implies order, such as needing to complete steps 1, 2, and 3 in the correct order. These nine features, however, don't need to be acknowledged in a certain order or be displayed in order of importance, right? If so, it's better to use an unordered or bulleted list instead.
- It's nice to make lists parallel, such as having all main points of an outline appear as a noun plus a prepositional phrase. In this case, I think it'd be good to make all the items in the list complete sentences.

Reply:
Understood but cannot implement in this content for now.